geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/EnvironmentCommand.scala

```diff
@@ -74,5 +108,26 @@ object EnvironmentCommand {
 
     @Parameter(names = Array("-c", "--converters"), description = "Examine GeoMesa converters", variableArity = true)
```

Comment:
can we change the doc for ```-c``` and ```-s``` here and above to make them:

```Describe a specific simple feature type``` and ```Describe a specific GeoMesa converter``` instead of being plural